example/otel-collector/otel-collector.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright The OpenTelemetry Authors
+# SPDX-License-Identifier: Apache-2.0
+
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:4317
+processors:
+extensions:
+  health_check: {}
+exporters:
+  otlp:
+    endpoint: jaeger:4317
+    tls:
+      insecure: true
+  prometheus:
+    endpoint: 0.0.0.0:9090
+    namespace: testapp
+  logging:
+
+service:
+  extensions: [health_check]
+  pipelines:
+    traces:
+      receivers: [otlp]
+      processors: []
+      exporters: [otlp]
```

Comment:
```suggestion
      exporters: [otlp, logging]
```

Reply:
From the doc of logging exporter, https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/loggingexporter
> This exporter is being deprecated in favour of the [debug exporter](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/debugexporter/README.md). It will be removed in September 2024.

We can remove the entire logging exporter.